lib/shiftregister.js

```diff
@@ -28,14 +36,34 @@ function ShiftRegister(opts) {
   });
 }
 
+/**
+ * Send one or more values to the shift register.
+ * @param {...number} value Value to send
+ * @returns {ShiftRegister}
+ */
 ShiftRegister.prototype.send = function(value) {
+  var args = Array.prototype.slice.apply(arguments);
   this.board.digitalWrite(this.pins.latch, this.io.LOW);
-  this.board.shiftOut(this.pins.data, this.pins.clock, true, value);
+  args.forEach(function(val) {
+    this.board.shiftOut(this.pins.data, this.pins.clock, true, val);
+  }, this);
   this.board.digitalWrite(this.pins.latch, this.io.HIGH);
 
-  priv.get(this).value = value;
+  priv.get(this).value = args.length > 1 ? args : value;
 
   return this;
 };
 
+/**
+ * Clear the shift register by replacing each value with a 0.
+ * @type {ShiftRegister}
+ */
+ShiftRegister.prototype.clear = function () {
+  var value = priv.get(this).value;
+  if (Array.isArray(value)) {
+    return this.send.apply(this, numericFill(value.length));
```

Comment:
Works for me. Since it's used to clear, you can just declare it once in the module scope and reuse it for all calls to `clear()`